homeassistant/components/alexa_devices/services.yaml

```diff
@@ -0,0 +1,506 @@
+send_text_command:
+  fields:
+    device_ids:
```

Comment:
I think this should be device_id (without s).
Without that, I get an error when I execute developer tools->actions->send_text_command:

Failed to perform the action alexa_devices.send_text_command. extra keys not allowed @ data['device_ids']. Got None required key not provided @ data['device_id']. Got None

Reply:
It allows multiple device ids, it's a list